File: serv/src/main/java/com/example/serv/models/PersonDataModel.java
package com.example.serv.models;

import lombok.AllArgsConstructor;

import java.io.Serializable;

@AllArgsConstructor
public class PersonDataModel implements Serializable {

    public int id;
    public String name;
    public String lastName;
    public String phone;
    public String email;
    public String bday;
    public int male;

}

File: app/src/main/java/com/example/lr2/datamodel/PersonDataModel.kt
package com.example.lr2.datamodel

/***
 * Класс с данными пользователя
 */
data class PersonDataModel(
    var id: Int,
    var name: String,
    var lastName: String,
    var phone: String,
    var email: String,
    var bday: String,
    var male: Int //1-fem, 0- man
) : java.io.Serializable